test/scripts/load-testing/pstatus_load_test.py

```diff
@@ -35,7 +35,7 @@ async def send_single_message(sender, message):
 async def run():
     # Generate a unqiue upload ID
     upload_id = str(uuid.uuid4())
-    dex_ingest_datetime = datetime.utcnow().replace(microsecond=0).isoformat() + 'Z'
+    dex_ingest_datetime = datetime.now(timezone.utc)
```

Comment:
We'd like the string output of this date/time to look like "2024-08-14T15:00:23Z".  In order to get this you need it to be:
```python
dex_ingest_datetime = datetime.now(timezone.utc).replace(microsecond=0).isoformat().replace("+00:00", "Z")
```
Otherwise, with the code now you get `2024-08-14 15:34:08.719713+00:00` which does pass validation.